src/ServiceControlInstaller.Engine/Instances/MonitoringTransports.cs

```diff
@@ -10,12 +10,26 @@ public class MonitoringTransports
         {
             new TransportInfo
             {
-                Name = "AzureServiceBus",
+                Name = "AzureServiceBus (Forwarding Topology)",
```

Comment:
Think we can skip the topology since the new one only have a single one?

Reply:
I'm not familiar with Azure, if users are aware that the default topology (and only one going forward) is the Forwarding one than yes I'd say it's redundant.

Reply:
@Particular/azure-maintainers thoughts on ☝️ ?

Reply:
If that's a Legacy one, shouldn't it say so? Otherwise, please keep it as "Azure Service Bus" (not one word).

Reply:
That's the new one @SeanFeldman, in this PR there are also the legacy ones. I kept the name one word to be consistent with what it was before. I'll change it for all the Azure ones.